integrations/Satismeter/browser.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-var */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import logger from "../../utils/logUtil";
+import { NAME } from "./constants";
+import { LOAD_ORIGIN } from "../ScriptLoader";
+
+class Satismeter {
+  constructor(config, analytics) {
+    logger.debug("in satismetwr const");
+    if (analytics.logLevel) {
+      logger.setLogLevel(analytics.logLevel);
+    }
+    this.name = NAME;
```

Comment:
We should use the new common name module. For reference: https://github.com/rudderlabs/rudder-sdk-js/pull/685, https://github.com/rudderlabs/rudder-sdk-js/pull/725

Reply:
Unable to understand this as the changes already were reverted at the destination m
odule level

Reply:
Unable to understand this as changes were in #725 at destination module level.

Reply:
Yes, I didn't know it was reverted. So no need to implement this.